cdap-ui/server/express.js

```diff
@@ -383,36 +391,14 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
     For now it handles file upload POST /namespaces/:namespace/apps API
   */
   app.post('/namespaces/:namespace/:path(*)', function(req, res) {
```

Comment:
With the proxy, we should refactor this out with all usage of API going through the `/api` proxy. Open a JIRA to not forgetting it.

Reply:
👍 Have filed this [JIRA - CDAP-15630](https://issues.cask.co/browse/CDAP-15630)